source/includes/harvest/_job_posts.md

```diff
@@ -169,7 +169,13 @@ curl 'https://harvest.greenhouse.io/v1/job_posts'
     "id": 123,
     "title": "Button Pusher",
     "location": {
-      "name": "The Island"
+        "id": 123456,
+        "name": "New York, NY",
+        "office_id": null,
+        "job_post_location_type": {
+            "id": 1,
+            "name": "Free Text"
```

Comment:
For simplicity, I've used free text location fields for all my examples. However, this may also be of type offices only or user defined list. 

For example, an office location would look like this:
```
"location": {
        "id": 1234,
        "name": "Astoria",
        "office_id": 2345,
        "job_post_location_type": {
            "id": 1123,
            "name": "Offices only"
        }
    },
```

Not sure if/how that merits calling out.